### PR TITLE
Adds the Cargo-Tech-of-the-Shift award

### PR DESCRIPTION
**Pull request by MrDoomBringer**
:cl: MrDoomBringer
add: Nanotrasen, as part of their new Employee Retention program, has encouraged more station point-makery by adding the "Cargo Tech of the Shift"! The award is located in a lockbox in the Quartermaster's locker.
/:cl:

Why: for too many years the cargotech has toiled under the unrewarding gaze of the quartermaster. Today marks the beginning of an era in which some will finally be recognized for their exemplary crate-dragging paper-pushing service. To remedy this, the Quartermaster has been given a lockbox with a single "Cargo Tech of the Shift" ribbon in it.

_(name reflects an "employee of the month" type aesthetic)
(name pending)_

Images:
image
image
image

-Added a ribbon object and a cargo-award as a subobject of ribbon in accessories.dm
-added a lockbox to the Quartermaster locker, populated with a single "cargotech of the shift" ribbon
-added icon to icons/mob/accessories.dmi
-added icon to icons/obj/clothing/accessories.dmi

because I'm expecting a high level of scrutiny over this game-changing pr, i've assembled a cogent and well-thought out Q&A

**Q1: why not give medal to hop?** 
A: because he is never around to truly admire the cargo tech's skills. _Further reasoning outlined in the last sentence of Question 3 part C_

**Q2: why a ribbon?**
A: this nuanced design choice is giving deference to the fact that the QM is not a true head (yet), and therefore centcomm has only supplied him or her with a single, cheap ribbon, instead of a few of those fancy-schmancy medals all the real heads get.
( I can make the award a medal if we really crave uniformity, though)
(this reason also reflects why I didnt want to call it the 'Cargo medal' or something)

**Q3: job-specific medals seem silly. why not have botany - (or any other civ. job) -specific medals?**
A: I will counter this reasonable query with a three-pronged attack:
1. cargo is currently the only job that has a leader position without medals (the Quartermaster). The role of a head is to manage, provide permission, and encourage their subordinates to do their best. The Quartermaster is already given tools to manage and provide permission (The Megaphone the Quartermaster's stamp, respectively) - so why not allow them to encourage their subordinates with certain incentives? This is the purpose of cargo's new award.

2. With other jobs (eg. botany, chef, janitor), the player typically sticks to their role and does their job. Unfortunately, many players treat cargotechs as assistants with extra access, therefore the ribbon can be used to incentivize cargotechs towards actually doing their job

3. Cargo shows the biggest upper limit for doing a great job- a fantastic botanist usually has deathnettles and some potent ambrosia, a fantastic chef will have a shitton of (admittedly varied and delicious) food. But a fantastic cargo tech? A fantastic cargo tech will be the most likely to turn a cargo bay into a heavily armed fortress of death powered by a double-singularity, guarded by a sizable armed militia of MULEBOTS-riding greyshits, and still make thousands of points to spare for the rest of the station. That kind of opportunity for greatness needs to be recognized and commended as such by the position of authority most likely to notice both the grand and nuanced activities of said cargotech- the quartermaster

**tldr: more items=more fun**

### CODE REVIEW

**Comment by praisenarsie** [comment]
ALRIGHT ALRIGHT YOU CAN HAVE IT

**Comment by MrDoomBringer** [comment]
image

cargo medals, or: how i begin to slowly turn qm into a real head

**Comment by praisenarsie** [comment]
that's up to headmins and design lead (who is also a headmin) for QM to become a head but the chances of QM becoming head is low unless hell freezes over.

**Comment by ExcessiveUseOfCobblestone** [comment]
@RemieRichards hehehe

**Comment by ExcessiveUseOfCobblestone** [comment]
Also the steelepoint of cargo???

**Comment by MrDoomBringer** [comment]
MERGE CONFLICTS ERADICATED LETS GO BOYS

**Comment by MrDoomBringer** [comment]
@praisenarsie just let me dream okay

**Comment by Anonmare** [comment]
Power creep

**Comment by ExcessiveUseOfCobblestone** [comment]
Also QM head will never happen

**Comment by MMMiracles** [comment]
QM will never be a head because they're deserving of a much higher position of power.

**Comment by GunHog** [comment]
This is neato, but Cargo IS a department, the "Supply" department. Do not let anyone tell you differently!

**Comment by Bawhoppen** [comment]
Was a wall of text truly necessary?

**Comment by MrDoomBringer** [comment]
@bawhoppen no
@GunHog oh yeah. well, rest of point still stands

**Comment by DoctoreMad** [comment]
Honestly I think this just further encourages HoPs to ignore their legitimate department in order to HoPcurity or sit around changing IDs(Which is important but also honestly boring and repetitive)
Side request, can you give the HoP a service ribbon equivalent which the botanist/bartender/chef/janitor/[non-supply civilian] all fight over for?

**Comment by DoctoreMad** [comment]
Nevermind how far removed some heads of staff are for their departments, HoP is just honestly the worst example.

**Comment by Bawhoppen** [comment]
What are you talking about? HOPs have tons of stuff to do. They're supposed to indirectly manage all of the station personnel, with the other heads managing them directly. That's why their main job is usually to change IDs and jobs. They're also the second in command of the station, serving directly to the captain.

**Comment by DoctoreMad** [comment]
>They're supposed to

Yeah and Wardens are supposed to guard the brig.

**Comment by Bawhoppen** [comment]
Which they do or get shunned by the populace

**Comment by DoctoreMad** [comment]
4aebc6fafd95fa54296911e1025b72254a899f6cb6536f0e64a618636d1177d4

**Comment by Bawhoppen** [comment]
From my understanding you were implying wardens didn't actually watch over the brig like they were supposed to, but I replied by stating they actually did in reality. What am I missing?

**Comment by DoctoreMad** [comment]
Ah fuck, I need to take this conversation seriously.
Well first problem is that I don't know how your first reply to me relates to what I said. Yeah its cool that HoPs still change IDs and succeed the Captain, but it should really be encouraged for them to care about Supply/Civilian positions before the Captain dies(If he even does), which was the main point.

So back to my post 1
My complaint is that the HoP should definitely get this badge, and not the QM. I'm really hoping HoPs would watch over their department(Supply and Civilian), which they don't really do anymore. Hell other servers have come up with inspectors and shit to watch over the civilian sector since after fucking years they think that role isn't filled.

**Comment by MrDoomBringer** [comment]
@DoctoreMad , There's pretty good reason for HoP not having this medal (beyond what was already outlined in the first question of the Q&A in the OP)

Your point of his job being boring or repetitive is valid, but having him award medals to cargo does not fix that at all. That implies that he would then spend time watching or observing cargotechs - meaning he ignores his actually important job (hopline being closed is infuriating when the hop's main job is to sit in hopline)

Though, hop spending a little time checking in on cargo wouldn't be so bad if a cargo tech makes a singulo or something, but that leaves out those nuanced cargotechs who are subtlety fantastic, such as by constantly bringing in crates or work keeping cargo in perfect order.

You hope that HoP should watch over their departments, but that is functionally impossible to do well (see above paragraph) without completely ignoring their main role (also if hop is gone from office for <2 seconds their room is getting fucking looted but that's a different issue). 

It seems like the solutions to your issue would be to either add more interesting features/shit to the hop role that he can perform inside his office, or add a new command role that is specifically made to observe and reward the crew (which takes away from the point of the heads)

**Comment by DoctoreMad** [comment]
​>Start making a post about ho HoPs main job is actually his two sectors
​>Realize that its been so much time and taking my sudden change of direction would ruin everything anyways
Shit

**Comment by ExcessiveUseOfCobblestone** [comment]
> You hope that HoP should watch over their departments, but that is functionally impossible to do well (see above paragraph) without completely ignoring their main role (also if hop is gone from office for <2 seconds their room is getting fucking looted but that's a different issue).

TFW basically sandbox game but people still think you can't move the map on a per-round basis to fit your needs.

TFW people don't know about the HOP tablet being a mobile ID changing device.

**Comment by MrDoomBringer** [comment]
@ExcessiveUseOfCobblestone 

> TFW basically sandbox game but people still think you can't move the map on a per-round basis to fit your needs.
ah im hop let me move my line into cargo??

>TFW people don't know about the HOP tablet being a mobile ID changing device.
ah i need an ID changed let me hunt down hop and ask him in the middle of a room

**Review by imtakingabreakdontatme — APPROVED** [review]
